go.mod

```diff
@@ -16,6 +15,7 @@ require (
 )
 
 require (
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
I mean it's still there 🙃

Reply:
Yea, there's some circular dependencies to cleanup.